src/main/kotlin/de/danielmaile/mpp/item/recipe/ShapedRecipe.kt

```diff
@@ -19,37 +19,151 @@ package de.danielmaile.mpp.item.recipe
 
 import org.bukkit.inventory.ItemStack
```

Comment:
Maybe add a comment that explains what this class does?

Reply:
Names are pretty descriptive, we could add comments inside of the getRecipes method to explain what each part of the code does though.

Reply:
The class is way more readable now ^^